superset-frontend/plugins/plugin-chart-table/test/TableChart.test.tsx

```diff
@@ -94,7 +94,7 @@ describe('plugin-chart-table', () => {
         <TableChart {...transformProps(testData.advanced)} sticky={false} />,
       );
       tree = wrap.render();
-      // should successful rerender with new props
+      // should successfully rerender with new props
```

Comment:
I mentioned I'd do this a while ago and one of the comments caused my reminder to reappear, so I've done it...